### PR TITLE
Bug 1812378 - Fix cookie_banner_exception_panel_description_state_off_for_site string

### PR DESCRIPTION
**Pull request by iorgamgabriel**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/main/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1812378

### CODE REVIEW

**Review by Delphine — APPROVED** [review]
Strings now look good to me, thank you

File: focus-android/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -693,7 +693,8 @@
     <string name="cookie_banner_exception_panel_description_state_on_for_site">%1$s will clear this site’s cookies and refresh the page. Clearing all cookies may sign you out or empty shopping carts.</string>
 
     <!-- Long text for a detail explanation indicating what will happen if cookie banner handling is off for a site, this is shown as part of the cookie banner panel in the toolbar. The placeholder will be updated with the app name. -->
-    <string name="cookie_banner_exception_panel_description_state_off_for_site">%1$s can try to automatically reject cookie requests.</string>
+    <string name="cookie_banner_exception_panel_description_state_off_for_site" moz:RemovedIn="111" tools:ignore="UnusedResources">%1$s can try to automatically reject cookie requests. If a reject option isn’t available, %1$s may accept all cookies to dismiss the banner.</string>
+    <string name="cookie_banner_exception_panel_description_state_off_for_site2">%1$s can try to automatically reject cookie requests.</string>
```
Comment (Amejia481) [inline focus-android/app/src/main/res/values/strings.xml:697]: I think we need to duplicate the comment, as the one above will be deleted when  we remove "cookie_banner_exception_panel_description_state_off_for_site".

**Review by Amejia481 — APPROVED** [review]
Looks good!
Let's just address this [comment](https://github.com/mozilla-mobile/firefox-android/pull/621/files#r1086989900) :)

File: focus-android/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (iorgamgabriel) [reply focus-android/app/src/main/res/values/strings.xml:699]: Done

**Pull request merged**